dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -206,6 +206,12 @@ http.proxyredirect.httpsPort.desc=The (secure) port to redirect to. Incoming HTT
 http.autoDecompression=Auto decompression
 http.autoDecompression.desc=Specifies whether the HTTP Channel automatically decompresses incoming request body data.
 
+http.decompressionRatioLimit=Decompression ratio limit
+http.decompressionRatioLimit.desc=Specifies the maximum ratio of decompressed to compressed request body payload. This ratio is verified as the body is read by the HTTP channel, and if the decompressionTolerance is reached, the request is ended.
+
+http.decompressionTolerance=Decompression tolerance
+http.decompressionTolerance.desc=Specifies the maximum number of times the HTTP channel can hit the decompressionRatioLimit as the request body is decompressed before the request is ended.
```

Comment:
Change,
"Specifies the maximum number of times the HTTP channel can hit the decompressionRatioLimit as the request body is decompressed before the request is ended."
to
"Specifies the maximum number of times the HTTP channel can reach the decompressionRatioLimit during request body decompression before the request ends."

Reply:
same here explain decompressionRatioLimit

Reply:
Updated the comment above.